addons/delivery/tests/test_delivery_stock_move.py

```diff
@@ -108,7 +108,7 @@ def test_02_delivery_stock_move(self):
             'product_id': self.product_cable_management_box.id,
             'company_id': self.env.company.id,
         } for x in range(5)])
- 
+
```

Comment:
You have to bring back the empty space if you want to avoid this diff :D

Reply:
Aaahhh, Im' trying but Pycharm does not let me. Gonna do it in terminal hahaha 
*keeping calm*

Reply:
Finally... haha